apps/web/pages/success.tsx

```diff
@@ -285,7 +285,7 @@ export default function Success(props: SuccessProps) {
       <BookingPageTagManager eventType={eventType} />
       <CustomBranding lightVal={props.profile.brandColor} darkVal={props.profile.darkBrandColor} />
       <main className={classNames(shouldAlignCentrally ? "mx-auto" : "", isEmbed ? "" : "max-w-3xl")}>
-        <div className={classNames("overflow-y-auto", isEmbed ? "" : "z-50 ")}>
+        <div className={classNames("overflow-y-auto sm:mx-auto sm:w-fit", isEmbed ? "" : "z-50 ")}>
```

Comment:
@harshsinghatz You might not need to change this styling. The way closing logic works is. It looks for an element having main class or a main tag itself. If click happens outside it, it closes the modal.

So, we need to appropriately place the `main` class on the element that's just filling the content.

I think the div element with classes "inline-block transform overflow-hidden rounded-lg" is the nearest. Try adding main class to it and fix any issue that come up after that.

Reply:
Yeah, just adding the main class worked!